test/emqx_cm_SUITE.erl

```diff
@@ -133,6 +133,9 @@ t_kick_session(_) ->
     ok = emqx_cm:unregister_channel(<<"clientid">>),
     ok = meck:unload(emqx_connection).
 
+t_all_channels(_) ->
+    ?assertEqual(true, is_list(emqx_cm:all_channels())).
```

Comment:
Add one more test case to verify the emqx_cm:all_channels/0 returns correct list as expected. i.e. check the length of the result is correct.